Spatial_searching/doc/Spatial_searching/CGAL/Kd_tree.h

```diff
@@ -96,19 +96,30 @@ template <class InputIterator> Kd_tree(InputIterator first, InputIterator beyond
 The constructor does not build the internal data structure, and it
 is also not updated after calls to `insert()`.  
 The method `build()` is called implicitly
-at the first call to a query member function. You can call
+at the first call to a query or removal member function. You can call
 `build()` explicitly to ensure that the next call to
 query functions will not trigger the reconstruction of the
 data structure.
 */
 void build();
+
+/*!
+This clears the internal data structure, which then gets rebuilt either by an
+explicit call to `build()` or implicitly by the next query or removal. The only
+reason to call this function explicitly is to rebalance the tree after some
+number of removals.
+*/
+void invalidate_built();
```

Comment:
I don't have a strong preference, we invalidate what was built, we invalidate the build, whatever. Maybe I was inspired by is_built(). Should I rename to invalidate_build?